src/man/sssd.conf.5.xml

```diff
@@ -1492,6 +1492,33 @@ pam_account_locked_message = Account locked, please contact help desk.
             <quote>[domain/<replaceable>NAME</replaceable>]</quote>
             <variablelist>
                 <varlistentry>
+                    <term>domain_type (string)</term>
+                    <listitem>
+                        <para>
+                            Specifies whether the domain is meant to be used
+                            by POSIX-aware clients such as the Name Service Switch
+                            or by applications that do not need POSIX data to be
+                            present or generated.
+                        </para>
```

Comment:
Maybe it should be said explicitly that one users and groups from POSIX domains are available in the OS.